src/headless/REQUIREMENTS.txt

```diff
@@ -0,0 +1,10 @@
+requests>=2.6.2
+tzlocal==1.2
+docopt
+celery
+SQLAlchemy
+nosexcover
+nose
+coverage
+redis
+flower
```

Comment:
We don't need anymore:

docopt
SQLAlchemy
redis
flower

Also, please pin the version, like this:

```
requests>=2.6.2
tzlocal==1.2
hammock==0.2.4
nose2==0.6.5
coverage==4.4.2
pyinotify==0.9.6
celery==4.1.0
mock==2.0.0
```

Reply:
Done.